src/binary/ibc-setup/commands/balances.ts

```diff
@@ -0,0 +1,86 @@
+import os from 'os';
+import path from 'path';
+
+import { stringToPath } from '@cosmjs/crypto';
+import { GasPrice } from '@cosmjs/launchpad';
+import { DirectSecp256k1HdWallet } from '@cosmjs/proto-signing';
+import { Coin } from '@cosmjs/proto-signing/build/codec/cosmos/base/v1beta1/coin';
```

Comment:
Looks like the `Coin` is not exported. Should I drop it?

Reply:
Oh, yeah don't use that one.
We should have the same coin in `../../../codec/cosmos/...`

But that's a proto type with no special functionality. If you need the interface for types, use the local one

Reply:
f9def0b